pkg/operator/controller/ttl_report_test.go

```diff
@@ -0,0 +1,28 @@
+package controller
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTTLIsExpired(t *testing.T) {
```

Comment:
Normally I would not buy these tests of a private method but it's okey for this merge request. A better practice for unit testing the TTL reconciler would be to test the `func (r *TTLReportReconciler) ReconcileReport() reconcile.Func` function, which is the only contract that we must obey.  The downside of testing private methods is that even if we don't change any core logic of the reconciler, changing just a private method would require rewriting unit tests. Such tests are not reliable and difficult to maintain. Some people even consider it an anti-pattern.